lang/fr-fr.js

```diff
@@ -40,7 +40,7 @@ export default {
 	"components.input-date-time-range.endDate": "Date de fin",
 	"components.input-date-time-range.errorBadInput": "{startLabel} doit être antérieur à {endLabel}",
 	"components.input-date-time-range.startDate": "Date de début",
-	"components.input-date-time.date": "Date", // mfv-translated
+	"components.input-date-time.date": "Date",
```

Comment:
Ugh, that sucks that it removes this. Will log this and see if the tooling can be updated.